packages/openapi-generator/src/wrapper-files/create-file.ts

```diff
@@ -41,3 +41,28 @@ ${content}
 ` + '\n'
   );
 }
+
+/**
+ * @experimental This API is experimental and might change in newer versions. Use with caution.
+ * Copy a file from a given path.
+ * @param src Path to the source file.
+ * @param dest Path to the destination file
+ * @param overwrite Whether or not existing files should be overwritten.
+ */
+export async function copyFile(
+  src: string,
+  dest: string,
+  overwrite: boolean
+): Promise<void> {
+  if (!overwrite && (await existsSync(dest))) {
+    return;
+  }
+  fsCopyFile(src, dest, err => {
```

Comment:
Isn't there a promises version of this function? With a Promise you can wait for the copy to finish - currently the end is sometime, but nobody knows when.

Reply:
👍 switched to the promises version